content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -3,135 +3,211 @@ title: Instrumentation
 weight: 3
 ---
 
-This guide will cover creating and annotating spans, creating and annotating metrics, how to pass context, and a guide to automatic instrumentation for JavaScript. This simple example works in the browser as well as with Node.js
+Manual instrumentation is the process of adding observability code to your
+application.
 
-## Example Application
+## Initialize Tracing
 
-In the following this guide will use the following sample app:
+To start [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry),
+you'll need to have an initialized
+[`TracerProvider`](/docs/concepts/signals/traces/#tracer-provider) that will let
+you create a [`Tracer`](/docs/concepts/signals/traces/#tracer).
 
-```javascript
-'use strict';
+If a `TracerProvider` is not created, the OpenTelemetry APIs for tracing will
+use a no-op implementation and fail to generate data.
 
-for (let i = 0; i < 10; i += 1) {
-  doWork();
-}
+### Node.js
+
+To initialize tracing with the Node.js SDK, first ensure you have the SDK
+package and OpenTelemetry API installed:
+
+```shell
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/sdk-trace-node
+```
+
+Next, create a separate `tracing.js` file that has all the SDK initialization
+code in it:
+
+```js
+const opentelemetry = require("@opentelemetry/api");
+const { NodeTracerProvider } = require("@opentelemetry/sdk-trace-node");
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+const { ConsoleSpanExporter, BatchSpanProcessor } = require("@opentelemetry/sdk-trace-base");
+
+const provider = new NodeTracerProvider();
```

Comment:
I would include a resource that has at least a service name since that is a required attribute